models/src/utils.py

```diff
@@ -285,17 +297,39 @@ def get_pods(model_name: str) -> List[DeployedModelPod]:
     return pods
 
 
+class NotConfiguredException(Exception):
+    pass
+
+
+def create_boto3_config():
+    boto3_config = {}
+    if S3_CREDENTIALS_PROVIDER == "minio":
+        logger_.info(f"S3_Credentials provider - {S3_CREDENTIALS_PROVIDER}")
+        boto3_config.update(
+            {
+                "aws_access_key_id": MINIO_ACCESS_KEY,
+                "aws_secret_access_key": MINIO_SECRET_KEY,
+                "endpoint_url": f"http://{MINIO_HOST}",
+            }
+        )
+    elif S3_CREDENTIALS_PROVIDER == "aws_iam":
+        logger_.info(f"S3_Credentials provider - {S3_CREDENTIALS_PROVIDER}")
```

Comment:
Let's move logging to a single line (L322)

Reply:
agreed